lib/base_service.js

```diff
@@ -270,17 +270,29 @@ BaseService.prototype._updateConfig = function _updateConfig(conf) {
     });
 };
 
-BaseService.prototype._requireModule = function(modName) {
+BaseService.prototype._requireModule = function(modName, opts) {
     var self = this;
+    opts = opts || { mod: modName, baseTried: false, modsTried: false };
```

Comment:
yes, because we pass in a different modName with the full path the second and third time around.